### PR TITLE
Add Paper to migrating from, Change Canary to CanaryMod

### PR DESCRIPTION
**Pull request by MrSlimeDiamond**

### CODE REVIEW

File: source/server/getting-started/migrating.rst [hunk 1]
```diff
@@ -127,7 +127,7 @@ When finding replacements for your Bukkit plugins, there are a few points to kee
 * Some Sponge plugins that are ported from Bukkit may change in functionality, or may not even use the same
   configuration structure.
 
-Migrating from Canary
+Migrating from CanaryMod
 ~~~~~~~~~~~~~~~~~~~~~
```
Comment (ST-DDT) [inline source/server/getting-started/migrating.rst]: Please extend the `~` line to the same length as the title right above it.

File: source/server/getting-started/migrating.rst [hunk 2]
```diff
@@ -20,7 +20,7 @@ Migrating to Sponge
 To migrate to SpongeForge or SpongeVanilla, look at the sections below. Most instructions are identical for both,
 differences are stated below.
 
-Migrating from CraftBukkit or Spigot
+Migrating from CraftBukkit, Spigot, or Paper
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (ST-DDT) [inline source/server/getting-started/migrating.rst]: Please extend the `~` line to the same length as the title right above it.

**Review by ST-DDT — CHANGES_REQUESTED** [review]

File: source/server/getting-started/migrating.rst (hunk 1 above)
Comment (MrSlimeDiamond) [reply source/server/getting-started/migrating.rst]: Okay. I did that

**Review by Inscrutable — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]
Thanks for your contribution.

**Pull request merged**